crates/fuzzing/src/generators/config.rs

```diff
@@ -144,6 +144,8 @@ impl Config {
             hogs_memory: _,
             nan_canonicalization: _,
             gc_types: _,
+            component_model_async_builtins: _,
+            component_model_async_stackful: _,
```

Comment:
To respond to the helpful CI reminder bot, I just want to point out that this change does not in fact exercise these new CM-level features.

Reply:
...but no longer: 326ed15.